KMD.Identity.TestApplications.SAML.MVCCore/appsettings.json

```diff
@@ -7,15 +7,35 @@
     }
   },
   "Saml2": {
-    "IdPMetadata": "https://identity.kmd.dk/federationmetadata/2007-06/federationmetadata.xml",
```

Comment:
Maybe we still want to keep this just out commented and with a note about when to use which IdPMetadata?

Reply:
ahh, good catch, I forgot to revert it :)